test/games/strategy/engine/data/UnitCollectionTest.java

```diff
@@ -8,43 +8,63 @@
 import java.util.Collection;
 import java.util.Iterator;
 
+import org.junit.Before;
 import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
 
 import games.strategy.triplea.delegate.Matches;
 import games.strategy.util.IntegerMap;
 
 public class UnitCollectionTest {
 
-  final GameData gameData = new MyGameData();
-  final UnitType unitTypeOne = new UnitType("Unit Type 1", gameData);
-  final UnitType unitTypeTwo = new UnitType("Unit Type 2", gameData);
-  final MyPlayerID defaultPlayerID = new MyPlayerID();
-  final PlayerID otherPlayerID = PlayerID.NULL_PLAYERID;
-  final UnitCollection uc = new UnitCollection(defaultPlayerID, gameData);
+  @Mock
+  private final GameData gameData = Mockito.mock(GameData.class);

```

Comment:
@Mock annotation handles the 'Mockito.mock' for you.  So this can be simplified to just:
```
@Mock
 private final GameData gameData;
```